[MRG] ENH added automatic conversion to bv for fif if no meg (#237)
mne-tools/mne-bids: README.rst

the following packages:

- ``nibabel``, for interacting with MRI data

> - ``pybv``, for converting FIF format EEG data to BV to be valid according to EEG BIDS specifications

Reviewer: ```suggestion
- ``pybv``, to convert EEG data to BrainVision if input format is not be valid according to EEG BIDS specifications
```

> - ``pybv``, to convert EEG data to BrainVision if input format is not valid according to EEG BIDS specifications

Reviewer 2: you'll need to rebase add a line to the list of dependencies

> - ``pybv``, to convert EEG data to BrainVision if input format is not be valid according to EEG BIDS specifications

Reviewer: ```suggestion
- ``pybv``, to convert EEG data to BrainVision if input format is not valid according to EEG BIDS specifications
```
